gapii/cc/runtime.cpp

```diff
@@ -54,13 +54,13 @@ void* gapil_encode_object(context* ctx, uint8_t is_group, uint32_t type,
   return nullptr;
 }
 
-void gapil_slice_encoded(context* ctx, slice_t* slice) {
+void gapil_slice_encoded(context* ctx, const void* slice) {
   DEBUG_PRINT("gapil_on_encode_slice(%p, %p)", ctx, slice);
```

Comment:
Just noticed that the DEBUG_PRINT here prints a different function name, might be good to fix with this CL or another one later.

Reply:
I'm removing the entire C API in following changes, so this will go away.